includes/blocks/class-sensei-reset-lesson-block.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * File containing the Sensei_Complete_Lesson_Block class.
+ *
+ * @package sensei
+ * @since 3.8.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Complete_Lesson_Block is responsible for rendering the 'Complete Lesson' block.
```

Comment:
Comments here need to be updated.

Reply:
Fixed in d6afb43efa324c5c58e005af615dfbf003fe3a89